vcpkg/get_started/get-started.md

```diff
@@ -59,6 +59,17 @@ This tutorial shows you how to create a C++ "Hello World" program that uses the
     > [!NOTE]
     > Setting the `VCPKG_ROOT` environment variable using the `set` command only affects the current shell session. To make this change permanent across sessions, you can use the `setx` command and restart the shell session.
 
+    ::: zone-end
+    ::: zone pivot="shell-powershell"
+
+    ```powershell
+    $env:VCPKG_ROOT = "C:\path\to\vcpkg"
+    $env:PATH = "$env:VCPKG_ROOT;$env:PATH"
+    ```
+
+    > [!NOTE]
+    > Setting the `VCPKG_ROOT` environment variable in this way only affects the current shell session. To make this change permanent across sessions, you would typically modify them through the Windows System Properties.
```

Comment:
I couldn't find what the Windows System Properties were and how to set them. Can you explain what that is?